Sources/EnhancedMediaPlayer/MediaPlayerControlsView.swift

```diff
@@ -94,6 +108,21 @@ struct MediaPlayerControlsView: View {
         }
     }
 
+    @ViewBuilder private func renderTimeBar(color: Color = .blue) -> some View {
```

Comment:
The ticket asks for `.primary` to be the default color, I believe this affects the behavior when switching from a light theme to a dark one

Reply:
`.primary` is not `.blue` and barely visible in light mode, so I decided to leave `.blue` for now since it works for both themes